### ISSUE TITLE
Make Drupal source of Truth for VBA facility data

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#15366]
## User Story or Problem Statement
The current source of truth for VBA facility data is Sandy's database, which is maintained by @mmiddaugh. Once we have VBA editors directly updating Facility information in Drupal, we will need to update migrations so that updates in Drupal are sent to the Facilities API

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] User support
- [ ] Accelerated Publishing

**Comment by swirtSJW** [comment]
This unfortunately has a lot of changes that have to be all done at once.  They can not be done  in smaller bites without use of an integration branch, which I do not think the size of this warrants.

**Comment by mmiddaugh** [comment]
**Update 6.7.2024**
- Pending Lighthouse capacity and Regional office modernization initiative

**Comment by davidconlon** [comment]
This issue is a duplicate of https://github.com/department-of-veterans-affairs/digital-experience-products/issues/304

Moving that information here for reference 

#  Create VBA Veteran-Facing Facility SSOT in Drupal
- The current VBA facility table maintained be Sandy Tadeo in the CDW is without a maintainer when Sandy Tadeo left VBA in April 2021 because it is maintained in SQL and no one in VBA Digital Media has those skills. 
- After VBA analyzed this data in conjunction with VBA Facilities and found the data to be inaccurate beyond the point of recovery.

This effort would entail partnering with OFO and VBA Digital Media to create a naming rubric, service taxonomy (https://github.com/department-of-veterans-affairs/digital-experience-products/issues/17), and system of record for Veteran-facing VBA facilities and their services with the intent being a system of record that VBA can then actively maintain with system support from CMS help desk and integration into the larger VA.gov ecosystem

By creating this system in Drupal, OCTO-DE can:
- Provide a UX to enable VBA Digital Media to continue to maintain this data
- Allow for OCTO-DE to have shape this data alongside #286 
- Have opportunities to re-align the data to integrate with the other administrations
- Develop relationships with new/more staff in VBA Digital Media and OFO
- Enable Drupal to be canonical information about VBA Veteran-facing facilities in-and-of itself
- Create foundational work for the future Regional Offices sites including relationships with the field
Drupal is already feeding facility status information to Lighthouse. this would be in a similar vein.
GeoBISL could draw data from either Drupal or Lighthouse for its needs
It's important to note that the alternative, GeoBISL, would be providing a similar web form to VBA to maintain this data.

## Headlines
**Business stakeholder -** VBA Digital Media, VBA Office of Field Operations, VBA Office of Mission Support

**DEPO owner -** Dave Conlon

**Is request tracked to an event or Congressional deadline? -** No

## Details
**Who is the primary user?** - Office of Field Operations, Lighthouse, GeoBISL, (Facility locator by way of lighthouse)

**What are the user goals for the project/page/product?** Maintain accurate information about VBA facilities including address, phone number, services

**What is the business goal?** Provide Single-source-of-truth information about VBA facilities.

**How does this map to our North Star?**

1. Increase availability of self-service tools - Foundational. Future will enable cross-linking services with online alternatives
2. Decrease time to outcomes - Medium + Foundational.  Data accuracy is terrible
3. Maximize satisfaction, reliability, availability, & security - High - Data is currently inaccurate with no clear way to update. Will enable data changes to re-align/minimize duplication and lower barrier to keep data up-to-date.

**How will we measure success?**

**How well do we understand the problem?**
Medium.
We understand most of the existing facility sub-types with the exception of "Outbased" which needs some better definition/guardrails put in-place.
We have a decent idea of what services are provided at each location in VA-language but not necessarily in Veteran-facing language.

**What is DEPO's role in this work?**
- Support OFO in developing facility type standardization/definitions
- Review data & re-structure as necessary
- Provide similar data feeds to Lighthouse/GeoBISL
- Training & support for VBA data maintenance users

**What are the known risks?**
- If we don't do this, we are continuing to surface incorrect facility locations in Lighthouse and Facility Locator to Veterans.
- We don't know what we don't know: Are there customers of this CDW data that we are unaware of that would make turning this system off a problem
- **Note:** We have agreed with Office of Field Operations that this data in Drupal will only be for Veteran-Facing facilities

**Reference links:**

**Comment by mmiddaugh** [comment]
**Update 8.8.2024**
no change

**Comment by mmiddaugh** [comment]
**Update 9.3.2024**
- the ability to move all VBA data into Drupal is currently blocked by Lighthouse. Moving this back to the backlog

**Comment by mmiddaugh** [comment]
**Update 2.15.2025**
- This remains blocked by Lighthouse capacity

**Comment by JonathanKatora** [comment]
**Update 4.16.26**
Met with LH team to discuss Drupal as SSOT for VBA facilities. This effort is not yet prioritized by LH team due to scope of work involved. LH team focused on performance & reliability before major changes to reload redesign work.
